F.FFFE
======================================================================
ERROR: test_list_int (__main__.Test06_DeleteContact)
*** Test06 *** GIVEN: filename t99.dat = {'7145551111': ['Steve', 'Jobs'], '5625553333': ['Bill', 'Gates']} AND contacts = Contacts(filename='t99.dat') *** FUNCTION CALL: r = contacts.delete_contact(id = '7145551111') *** EXPECT: r = 'error' ***
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\edmar\OneDrive\Documents\CSUF Fall 2024\Python\Assignments\Assignment 7\lab07-master\lab07-master\test.py", line 95, in test_list_int
    r = contacts.delete_contact(id = '7145559999')
  File "C:\Users\edmar\OneDrive\Documents\CSUF Fall 2024\Python\Assignments\Assignment 7\lab07-master\lab07-master\contacts.py", line 37, in delete_contact
    remove = self.data.pop(id)
KeyError: '7145559999'

======================================================================
FAIL: test_list_int (__main__.Test01_AddContact)
*** Test01 *** GIVEN: filename t99.dat = {'7145551111': ['Steve', 'Jobs'], '5625553333': ['Bill', 'Gates']} AND contacts = Contacts(filename='t99.dat') *** FUNCTION CALL: contacts.add_contact(id = '7145551212', first_name = 'Richard', last_name = 'Stallman') *** EXPECT: filename t99.dat = {'7145551111': ['Steve', 'Jobs'], '5625553333': ['Bill', 'Gates'], '7145551212': ['Richard', 'Stallman']} ***
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\edmar\OneDrive\Documents\CSUF Fall 2024\Python\Assignments\Assignment 7\lab07-master\lab07-master\test.py", line 25, in test_list_int
    self.assertEqual(c, {'7145551111': ['Steve', 'Jobs'], '5625553333': ['Bill', 'Gates'], '7145551212': ['Richard', 'Stallman']})
AssertionError: {'714[71 chars]12': {'5625553333': ['Bill', 'Gates'], '714555[60 chars]n']}} != {'714[71 chars]12': ['Richard', 'Stallman']}
  {'5625553333': ['Bill', 'Gates'],
   '7145551111': ['Steve', 'Jobs'],
-  '7145551212': {'5625553333': ['Bill', 'Gates'],
-                 '7145551111': ['Steve', 'Jobs'],
-                 '7145551212': ['Richard', 'Stallman']}}
? ---------------                                       -

+  '7145551212': ['Richard', 'Stallman']}

======================================================================
FAIL: test_list_int (__main__.Test03_ModifyContact)
*** Test03 *** GIVEN: filename t99.dat = {'7145551111': ['Steve', 'Jobs'], '5625553333': ['Bill', 'Gates']} AND contacts = Contacts(filename='t99.dat') *** FUNCTION CALL: contacts.modify_contact(id = '7145551111', first_name = 'Richard', last_name = 'Stallman') *** EXPECT: filename t99.dat = {'5625553333': ['Bill', 'Gates'], '7145551111': ['Richard', 'Stallman']} ***
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\edmar\OneDrive\Documents\CSUF Fall 2024\Python\Assignments\Assignment 7\lab07-master\lab07-master\test.py", line 54, in test_list_int
    self.assertEqual(c, {'5625553333': ['Bill', 'Gates'], '7145551111': ['Richard', 'Stallman']})
AssertionError: {'7145551111': ['Steve', 'Jobs'], '5625553333': ['Bill', 'Gates']} != {'5625553333': ['Bill', 'Gates'], '7145551111': ['Richard', 'Stallman']}
- {'5625553333': ['Bill', 'Gates'], '7145551111': ['Steve', 'Jobs']}
?                                                     ^^^^^^^^^^^

+ {'5625553333': ['Bill', 'Gates'], '7145551111': ['Richard', 'Stallman']}
?                                                  +++++++++++   ^^^^^^


======================================================================
FAIL: test_list_int (__main__.Test04_ModifyContact)
*** Test04 *** GIVEN: filename t99.dat = {'7145551111': ['Steve', 'Jobs'], '5625553333': ['Bill', 'Gates']} AND contacts = Contacts(filename='t99.dat') *** FUNCTION CALL: r = contacts.modify_contact(id = '7145559999', first_name = 'Richard', last_name = 'Stallman') *** EXPECT: r = 'error' ***
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\edmar\OneDrive\Documents\CSUF Fall 2024\Python\Assignments\Assignment 7\lab07-master\lab07-master\test.py", line 67, in test_list_int
    self.assertEqual(r, 'error')
AssertionError: {'7145559999': ['Richard', 'Stallman']} != 'error'

======================================================================
FAIL: test_list_int (__main__.Test05_DeleteContact)
*** Test05 *** GIVEN: filename t99.dat = {'7145551111': ['Steve', 'Jobs'], '5625553333': ['Bill', 'Gates']} AND contacts = Contacts(filename='t99.dat') *** FUNCTION CALL: contacts.delete_contact(id = '7145551111') *** EXPECT: filename t99.dat = {'5625553333': ['Bill', 'Gates']} ***
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\edmar\OneDrive\Documents\CSUF Fall 2024\Python\Assignments\Assignment 7\lab07-master\lab07-master\test.py", line 83, in test_list_int
    self.assertEqual(c, {'5625553333': ['Bill', 'Gates']})
AssertionError: {'7145551111': ['Steve', 'Jobs'], '5625553333': ['Bill', 'Gates']} != {'5625553333': ['Bill', 'Gates']}
- {'5625553333': ['Bill', 'Gates'], '7145551111': ['Steve', 'Jobs']}
+ {'5625553333': ['Bill', 'Gates']}

----------------------------------------------------------------------
Ran 6 tests in 0.013s

FAILED (failures=4, errors=1)
